### PR TITLE
feat(Logs): Obfuscation UI

### PR DESCRIPTION
**Pull request by barbnewrelic**
Minor edits before FS for DOC-7857

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/logs/ui-data/obfuscation-ui.mdx
Comment (paperclypse) [inline src/content/docs/logs/ui-data/obfuscation-ui.mdx:311]: @barbnewrelic I feel a little uneasy about "easily" :) But maybe it really is?

**Review by paperclypse — DISMISSED** [review]
Looks good to me @barbnewrelic. Feel free to merge away!

**Comment by barbnewrelic** [comment]
Discussed with peer reviewer. No need for second review.

**Pull request merged**
